lang/eo.yml

```diff
@@ -1,10 +1,4 @@
 eo:
-  EditableFormField:
```

Comment:
This is deleting translations

We should probably merge this first https://github.com/silverstripe/silverstripe-userforms/pull/1076

Then redo `tx pull` and redo the PR